webpack/InsightsCloudSync/Components/InsightsTable/SelectAllAlert.js

```diff
@@ -25,7 +24,7 @@ const SelectAllAlert = ({
         )}
         actionLinks={
           <AlertActionLink onClick={selectAll}>
-            {sprintf('Select all %s recommendations', itemCount)}
+            {__('Select recommendations from all pages')}
```

Comment:
We decided to drop the actual number of all recommendations, also on other strings here
as we can't really tell if they are selectable.

also, for the `selectAll` functionality which updates the store with the `flag`,
the exact number doesn't really matter.